src/EventStore.ClientAPI/Delegates.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading.Tasks;
```

Comment:
This file can safely be removed, as well as the reference to it in the ClientAPI.